publish/publish.proj

```diff
@@ -65,6 +65,14 @@
                    Condition="'@(_MissingBlobNames)' == ''" />
     <Message Importance="High" Condition="'@(_MissingBlobNames)' != ''" Text="Skipping finalization, missing these blobs: '@(_MissingBlobNames)'" />
   </Target>
+  
+  <Target Name="UpdateVersionsRepo"
```

Comment:
Then manually run this command temporarily. I would much prefer we spend our energy on getting this done right the first time instead of doing again later this week.